### PR TITLE
Encoding is UTF-8, and codepoints higher than 127 are disallowed

### PR DESCRIPTION
**Pull request by JervenBolleman**

### CODE REVIEW

**Review by sjackman — APPROVED** [review]

File: GFA1.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The master version of this document can be found at
 
 The purpose of the GFA format is to capture sequence graphs as the product of an assembly, a representation of variation in genomes, splice graphs in genes, or even overlap between reads from long-read sequencing technology.
 
-The GFA format is a tab-delimited text format for describing a set of sequences and their overlap. The first field of the line identifies the type of the line. Header lines start with `H`. Segment lines start with `S`. Link lines start with `L`. A containment line starts with `C`. A path line starts with `P`.
+The GFA format is a tab-delimited text format for describing a set of sequences and their overlap. The first field of the line identifies the type of the line. Header lines start with `H`. Segment lines start with `S`. Link lines start with `L`. A containment line starts with `C`. A path line starts with `P`. The text is encoded in UTF-8 but is not allowed to use a codepoint value higher than 127.
```
Comment (sjackman) [inline GFA1.md:14]:
```suggestion
The GFA format is a tab-delimited text format for describing a set of sequences and their overlap. The text is encoded in UTF-8 but is not allowed to use a codepoint value higher than 127. The first field of the line identifies the type of the line. Header lines start with `H`. Segment lines start with `S`. Link lines start with `L`. A containment line starts with `C`. A path line starts with `P`.
```

File: GFA2.md [hunk 2]
```diff
@@ -99,6 +99,8 @@ almost all references to identifiers are oriented, by virtue of a post-fix + or
 A +-sign indicates the object is in the orientation it was defined, and a --sign indicates
 it should be reverse-complemented.
 
+A GFA2 file must be encoded in UTF-8 and is not allowed to use a codepoint value higher than 127.
+
```
Comment (sjackman) [inline GFA2.md:103]: Please append this sentence to the paragraph on line 82.

**Review by sjackman — CHANGES_REQUESTED** [review]
Thank you for this clarification! I've suggested moving the location of the text. The text itself is great.

**Comment by JervenBolleman** [comment]
> Thank you for this clarification! I've suggested moving the location of the text. The text itself is great.

Text is moved.

**Review by sjackman — APPROVED** [review]
Thank you for your first contribution to GFA-spec, Jerven! 🌟

**Pull request merged**